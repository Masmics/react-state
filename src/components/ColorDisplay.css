.ColorDisplay {
  display: flex;
  width: '5em';
  height: 5em;
    /* div {
      display: flex;
      width: 5em;
      height: 5em;
    } */
}
