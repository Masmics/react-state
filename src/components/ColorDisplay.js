import React from 'react';
import PropTypes from 'prop-types';
import styles from './ColorDisplay.css';

function ColorDisplay({ colorDisp }){
  return (
    <div className={styles.ColorDisplay} style={{ backgroundColor: colorDisp }}>div> 
  );
}

ColorDisplay.propTypes = {
  colorDisp: PropTypes.string.isRequired
};

export default ColorDisplay;

