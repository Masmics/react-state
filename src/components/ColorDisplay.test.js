import React from 'react';
import { shallow } from 'enzyme';
import ColorDisplay from './ColorDisplay';

describe('Color', () => {
  it('renders the color display', () => {
    const wrapper = shallow(<ColorDisplay color="red" />);
    expect(wrapper).toMatchSnapshot();
  });
});

