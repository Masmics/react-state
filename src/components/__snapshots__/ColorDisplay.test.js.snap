// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Color renders the color display 1`] = `
<div
  className="ColorDisplay"
  style={
    Object {
      "backgroundColor": "red",
    }
  }
>
  <Component />
  !
</div>
`;
